src/Client.php

```diff
@@ -201,6 +218,7 @@ protected function restCall($apiUrl, $verb, $guzzleParams, $apiversion='',$heade
         
         // Contact the server
         try {
+            $logModel-> setStartTime(microtime(true));
```

Comment:
If the call to populateRequestInfo is put directly preceeding $this->client->request, you can interalize the start time to the LogInformation class and remove the setStartTime function altogether

Reply:
This was done purposefully as I just wanted to have the actual time taken in the rest call and not the entire method execution time here. This is the same way we implemented in java as well.